third-party/libfabric/README

```diff
@@ -0,0 +1,36 @@
+===========================
+libfabric README for Chapel
+===========================
+
+This copy of libfabric 1.10.1 is being released with Chapel for
+convenience and was obtained from:
+
+  https://github.com/ofiwg/libfabric/releases/tag/v1.10.1
+    (the libfabric-*.tar.bz2 release tarball, not the source snapshot)
```

Comment:
I don't think we need to be this specific. I would follow other 3p READMEs and just do:


```
This copy of libfabric 1.10.1 is being released with Chapel for
convenience and was obtained from:

    https://github.com/ofiwg/libfabric/
```

Reply:
How do people know what specifically to pick up, if that's not there?  Is it common knowledge that in a github repo one should go to the `releases` tab and download the distribution tarball rather than the source snapshot?

Reply:
I'm not sure who "people" is in this case. I think that's common knowledge for people using github, especially somebody who would be upgrading third-party packages. I can't imagine anybody beyond the person upgrading third-party packages caring about where the package was obtained from.

Reply:
Actually, in the interest of uniformity I'll just make this change.